src/Diagnostics/define_diagnostics.jl

```diff
@@ -0,0 +1,156 @@
+"""
+    define_diagnostics!(land_model)
+
+Calls add_diagnostic_viariable! for all variables
+"""
+function define_diagnostics!(land_model)
+
+    # Stored in p
+
+    # Albedo
+    add_diagnostic_variable!(
+        short_name = "alpha",
+        long_name = "Albedo",
+        standard_name = "albedo",
+        units = "",
+        comments = "The fraction of incoming radiation reflected by the land surface.",
+        compute! = (out, Y, p, t) -> compute_albedo!(out, Y, p, t, land_model),
+    )
+
+    # Net radiation
+    add_diagnostic_variable!(
+        short_name = "rn",
+        long_name = "Net Radiation",
+        standard_name = "net_radiation",
+        units = "W m^-2",
+        comments = "Balance between incoming and outgoing shortwave and longwave radiation at the land surface.",
+        compute! = (out, Y, p, t) ->
+            compute_net_radiation!(out, Y, p, t, land_model),
+    )
+
+    # Surface temperature 
+    add_diagnostic_variable!(
+        short_name = "tsfc",
+        long_name = "Surface Temperature",
+        standard_name = "surface_temperature",
+        units = "K",
+        comments = "Temperature at the land-atmosphere interface.",
```

Comment:
How is T_sfc defined? In the bucket, I think we use the value at the top center. that might be better described as "Temperature of the land surface". But if this is defined as a skin temperature (or something), what you have is OK.

Reply:
I will leave this comment up for now. We should decide soon. What do you think @braghiere ?

Reply:
I agree with @kmdeck. Usually surface temperature refers to skin temperature (a combination of soil + canopy). Temperature at the land-atmosphere interface sounds more like skin temperature to me. How is this term calculated? What does this actually physically represent?

Reply:
I believe this is the temperature at the center of the top layer, as Kat explained, soil layers being discretized to n layers, n defined by users. We have T_sfc stored for the bucket, but no T_sfc for soilcanopymodel, at the moment, I believe (maybe this has changed). We could rename T_sfc to T_skin. (we have T at all soil layers for both models, but T_sfc only for bucket)

We should meet, the 3 of us, to discuss ClimaLand diagnostic variables. 
Sometimes, we compute the same things across models (bucket, soilcanopymodel, other), they will be stored differently (e.g., p.soil.T or Y.bucket.T), but will have the same name, units, comments.

Before we merge this PR, it will be good to go through the list of variables, and make sure we agree on all the naming (short long standard), units, and comments. I will have something for all, but we should make sure we all agree and it looks good together.

Reply:
Sounds good to me.